src/main/jte/bookings/single.jte

```diff
@@ -66,14 +75,14 @@
     <div class="card-actions justify-end">
         @if (allowEditing)
             @template.util.form(csrf = csrf, action = "/" + booking.getRoom().getId() + "/bookings/" + booking.getId(), method = DELETE, content = @`
-                <button type="submit" class="btn btn-ghost">
+                <button type="submit" class="btn btn-ghost" aria-label="${context.localize("bookings.tooltip.delete")}" title="${context.localize("bookings.tooltip.delete")}">
```

Comment:
`bookings.delete.tooltip` or similar would be nicer.
Also, don't repeat it for both `aria-label` and `title`